web-transport-quinn/src/session.rs

```diff
@@ -166,7 +166,14 @@ impl Session {
             self.conn
                 .accept_bi()
                 .await
-                .map(|(send, recv)| (SendStream::new(send), RecvStream::new(recv)))
+                .map(|(send, recv)| {
+                    debug_assert_eq!(send.id(), recv.id());
+                    let id = Arc::new(send.id());
```

Comment:
Alternatively, increment an atomic ID here and call it `local_id`. It would no longer have gaps, but it would be increment in receive order (not send order) so it wouldn't match on both ends.

I think `quic_id()` is more useful though.